src/mitol/google_sheets_refunds/api.py

```diff
@@ -30,6 +33,23 @@ def __init__(self):
             request_model_cls=RefundRequest,
         )
 
+    def is_configured(self):
+        """
+        Checks for required settings.
+
+        Returns:
+            bool: False if required settings are missing
+        """
+        missing = []
+        for variable in REQUIRED_GOOGLE_SHEETS_REFUNDS_SETTINGS:
+            try:
+                if not getattr(settings, variable):
+                    missing.append(variable)
+            except AttributeError:
+                missing.append(variable)
+                log.exception(f"{variable} is not set.")
+        return not missing and super().is_configured()
```

Comment:
I'd probably split this up into two function so you don't have to have this logic both here and in the base class: and overridden `get_required_settings()` and then just a single `is_configured()` method on the base class.